### PR TITLE
gcc: version bumped to 6.1.0.

### PR DESCRIPTION
**Pull request by Florin65**
For testing purposes only!
Some modules does not compile with it (most notably grub module).

### CODE REVIEW

**Comment by lunar-linux-b01** [comment]
SUCCESS!

File: compilers/gcc/BUILD [hunk 1]
```diff
@@ -1,35 +1,43 @@
-
 # using -pipe causes spurious test-suite failures
-bad_flags -pipe &&
+CFLAGS=${CFLAGS/-pipe/} &&
+CXXFLAGS=${CXXFLAGS/-pipe/} &&
```
Comment (v4hn) [inline compilers/gcc/BUILD]: `bad_flags` is standard in lunar and it removes *all* occurrences of the flag.
If `CFLAGS` contains the flag more than once, your code fails.

Please just rever this part.

File: compilers/gcc/BUILD [hunk 2]
```diff
@@ -39,12 +47,13 @@ install -d /usr/share/gdb/auto-load &&
...
 
 # POSIX conformance launcher scripts for c89 and c99
 install -m0755 {${SCRIPT_DIRECTORY},/usr/bin}/c89 &&
 install -m0755 {${SCRIPT_DIRECTORY},/usr/bin}/c99 &&
 
 # remove the offending broken freetype link
-rm -f $MODULE_PREFIX/lib/gcc/$BUILD/$VERSION/include-fixed/freetype
+rm -f /usr/lib/gcc/$BUILD/$VERSION/include-fixed/freetype
+
+#ln -sf /usr/bin/cpp /lib/cpp
```
Comment (v4hn) [inline compilers/gcc/BUILD]: Why did you remove `/lib/cpp`?

**Comment by v4hn** [comment]
Thanks for the request!
I'll definitely test it when I find the time.

**Comment by lunar-linux-b01** [comment]
SUCCESS!
